### DISCUSSION TITLE
Finnish Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#547] [Language Leaders]
Start discussion for Finnish (`fi`) language leader.

**Comment by GileanEU** [comment]
Anyone? I am happy to help but probably don't have time to be a language leader.

**Reply by huusissa** [reply]
Hello, i could be that one.

**Reply by synesthesiam** [reply]
Thanks @huusissa! Checking out your PR now.

**Reply by synesthesiam** [reply]
Just need a few tests and we're good to go :+1:

**Reply by huusissa** [reply]
i made some changes and all tested.

**Reply by huusissa** [reply]
well, still missing that fi folder in main. i forked and try my self. I dont wanna do all by my self ;)

**Reply by synesthesiam** [reply]
Sorry for the delay! Looks like @balloob helped get it merged. Adding you as language leader.

**Comment by Arkkimaagi** [comment]
@huusissa, if you're busy, I could take over as language leader for Finnish?

I have bunch of stuff [coming to Finnish](https://github.com/home-assistant/intents/compare/main...Arkkimaagi:home-assistant-intents:finnish_major_update), but my [first commit](https://github.com/home-assistant/intents/pull/903) from a few weeks back is still waiting for any reply from you or anyone.

**Reply by synesthesiam** [reply]
@Arkkimaagi adding you as a language leader

**Comment by synesthesiam** [comment]
@huusissa Since you haven't been active for a while, I'm going to remove you as a language leader. If you have time again in the future, let me know.
